src/apiCalls.js

```diff
@@ -85,7 +84,8 @@ const patchHits = (recipe) => {
       getRecipes()
         .then(res => res.json())
         .then(data => {
-          pageData.allRecipes = data.recipes;
+          const updatedRecipeHits = data.recipes.find(item => item.id === recipe.id).hits;
+          recipe.hits = updatedRecipeHits;
```

Comment:
The reason your graph does not update is because this is not updating the recipe from page data all recipes, this is updating the hits on current recipe (the card that changes constantly)

Reply:
replacing it with this would update the actual recipe's hits if you don't want to update all of page data here

`pageData.allRecipes.find(item => item.id === recipe.id).hits = data.recipes.find(item => item.id === recipe.id).hits`